packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1637,6 +1643,10 @@ class Padding extends SingleChildRenderObjectWidget {
 /// corner of the [Container] in difference to the center-oriented system used in
 /// the example above with [Alignment].
 ///
+/// ![A blue square container with the Flutter logo positioned according to the
+/// following FractionalOffset. A point is marked at the top left corner of the
```

Comment:
same question about "following".

Reply:
Nice catch. Thanks! I just copied the text over when I changed the layout.